docs/source/features.rst

```diff
@@ -277,34 +277,12 @@ Entropy Limiting
 
 Entropy scans can produce a high number of false positives such as git SHAs or md5
 digests. To avoid these false positives, enable ``exclude-entropy-patterns``. Exclusions
-apply to any strings flagged by entropy checks.
+apply to any strings flagged by entropy checks. This option is not available on the command line,
+and must be specified in your config file.
 
-For example, if ``docs/README.md`` contains a git SHA, this would be flagged by entropy.
-To exclude this, add ``docs/.*\.md$::^[a-zA-Z0-9]{40}$`` to ``exclude-entropy-patterns``.
-
-.. code-block:: sh
-
-    > tartufo ... --exclude-entropy-patterns "docs/.*\.md$::^[a-zA-Z0-9]{40}$"
-
-.. code-block:: toml
-
-    [tool.tartufo]
-    exclude-entropy-patterns = [
-      # format: "{file regex}::{entropy pattern}"
-      "docs/.*\.md$::^[a-zA-Z0-9]{40}$", # exclude all git SHAs in the docs directory
-    ]
-
-.. warning::
-    .. versionchanged:: 2.9.0
-    As of version 2.9.0, the above specification style has been deprecated, and
-    will be removed in version 3.0. The new style uses a TOML `array of tables`_
-    as shown below.
-
-    Note that this new syntax is not available on the command line, and must be
-    specified in your config file.
-
-Here is an example of how you might exclude SHA hashes in your docs, as well as
-hashes for GitHub Actions in your workflows:
+For example, if ``docs/README.md`` contains a git SHA and `.github/workflows/*.yml` containes pinned git SHAs
+this would be flagged by entropy.
+To exclude this, add an entry to ``exclude-entropy-patterns`` in the config file.
 
 .. code-block:: toml
 
```

Comment:
We should add documentation of all the available keys, which ones are optional, and what they default to.

Reply:
I had added these but they were reverted back in one of the rebases. Added them back now.